src/main/java/bio/terra/common/CloudPlatformWrapper.java

```diff
@@ -33,6 +33,10 @@ public static CloudPlatformWrapper of(CloudPlatform cloudPlatform) {
         }
     }
 
+    public boolean is(CloudPlatform cloudPlatform) {
```

Comment:
If we have this, then I think we can get rid of `isGcp()` and `isAzure()`. Doesn't seem useful to have them both. Another way to express this is `CloudPlatform getPlatform();` and a caller would write `wrapper.get() == CloudPlatform.GCP`. I guess it depends on what the common use cases are.